recipes-app/iot2050-firmware-update/files/iot2050-firmware-update.tmpl

```diff
@@ -121,6 +122,13 @@ class UpgradeError(Exception):
     def __str__(self):
         return self.erroinfo
 
+class ErrorCode(Enum):
+    SUCCESS = 0
+    INVALIDARG  = 1
+    CANCELED  = 2
+    BACKUPFAILED  = 3
+    ROLLBACKFAILED = 4
+    INTERNALERR = 5
```

Comment:
Better to use `_` to make it more clear, such as `INVALID_ARG`, `ROLLBACK_FAILED`